zip-0000.rst

```diff
@@ -163,51 +166,91 @@ SHOULD be added to the ZIP metadata indicating the original authorship
 ZIP Editors
 -----------
 
-The current ZIP Editors are Daira Emma Hopwood, representing the Electric Coin
-Company, and Deirdre Connolly, representing the Zcash Foundation. Both
-can be reached at zips@z.cash . The current design of the ZIP Process
-dictates that there are always at least two ZIP Editors: one from the
-Electric Coin Company and one from the Zcash Foundation. Additional Editors may
-be selected by consensus among the current Editors.
+The current ZIP Editors are:
+- Daira Emma Hopwood and Jack Grigg (representing the Electric Coin
+Company)
+- teor and Conrado Gouvea (representing the Zcash Foundation).
 
+All can be reached at zips@z.cash. The current design of the ZIP Process
+dictates that there are always at least two ZIP Editors: at least one from the
+Electric Coin Company and at least one from the Zcash Foundation.
 
-ZIP Editor Responsibilities & Workflow
---------------------------------------
+Additional Editors may be selected by consensus among the current Editors.
+An Editor may be removed or replaced by consensus among the current Editors.
+However, if the other ZIP editors have consensus, an Editor can not prevent their own removal.
+Any Editor can resign by informing the other Editors in writing.
+
+Whenever the ZIP editors change, the new ZIP editors should:
+- review this ZIP to make sure it reflects current practice,
+- update the owners of this ZIP,
+- review ZIP editor access to the `ZIPs repository <https://github.com/zcash/zips>`,
+- update the <zips@z.cash> email alias, and
+- invite new editors to the Zcash Community Forums, and the #zips channel on Discord.
+
+If it has been at least 12 months since the last ZIP editor change, the ZIP editors should:
+- review this ZIP to make sure it reflects current practice.
+
+
+ZIP Editor Responsibilities
+---------------------------
 
 The ZIP Editors subscribe to the `Zcash Community Forum.
 <https://forum.zcashcommunity.com/>`__
 
-For each new ZIP that comes in, a ZIP Editor confirms the following:
+Each new ZIP SHOULD be submitted as a "pull request" to the ZIPs git
+repository [#zips-repo]_. It SHOULD NOT contain a ZIP number unless
+one had already been assigned by the ZIP Editors. The pull request
+SHOULD initially be marked as a Draft. The ZIP content SHOULD be
+submitted in RST format. Generating HTML for a ZIP is optional.
```

Comment:
This change wasn't discussed last week, but it reflects current practice: ZIPs are submitted to both the forums and the git repository before ZIP Editors review them.